src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -390,37 +390,41 @@ QueryNetworkResult Plugin::QueryNetwork(const CNNNetwork& network,
     std::vector<std::shared_ptr<ngraph::Node>> nextLayerDependent;
 
     auto layerIsSupported = [&](std::shared_ptr<ngraph::Node> node) {
+        if (node->is_dynamic()) {
```

Comment:
what's about dynamic batch?

Reply:
I suggest merging it as is and address dynamic batch case in #9314